src/datasources/email/email.datasource.ts

```diff
@@ -75,12 +83,24 @@ export class EmailDataSource implements IEmailDataSource {
     account: string;
     code: string;
     codeGenerationDate: Date;
+    unsubscriptionToken: string;
   }): Promise<void> {
-    await this.sql<Email[]>`
+    const [email] = await this.sql<Email[]>`
         INSERT INTO emails.account_emails (chain_id, email_address, safe_address, account, verification_code,
-                                           verification_code_generated_on)
+                                           verification_code_generated_on, unsubscription_token)
         VALUES (${args.chainId}, ${args.emailAddress.value}, ${args.safeAddress}, ${args.account}, ${args.code},
-                ${args.codeGenerationDate})
+                ${args.codeGenerationDate}, ${args.unsubscriptionToken})
+        RETURNING *
+    `;
+
+    // When registering a new email address, the account_recovery subscription should be enabled by default
+    await this.sql`
+        INSERT INTO emails.account_subscriptions (account_id, subscription_id)
+        SELECT ae.id, s.id
```

Comment:
Nit: please resolve if I'm not understanding correctly but are `ae` and `s` self-define variables? If so, could we name them clearer?

Reply:
They are aliases to the table names (they come from the `FROM` clause). I will make the variables clearer!